File: solutions/2650-split-with-minimum-sum/solution.py
class Solution(object):
    def splitNum(self, num):
        """
        :type num: int
        :rtype: int
        """
        n=sorted(list(str(num)))
        a=""
        b=""
        for i in range(0,len(n),2):
            a+=n[i]
        for i in range(1,len(n),2):
            b+=n[i]
        return int(a)+int(b)

File: solutions/2650-split-with-minimum-sum/solution.java
class Solution {
    public int splitNum(int num) {
        String a="",b="";
        char[] arr=String.valueOf(num).toCharArray();
        Arrays.sort(arr);
        for(int i=0;i<arr.length;i+=2)
            a+=arr[i];
        for(int i=1;i<arr.length;i+=2)
            b+=arr[i];
        return Integer.parseInt(a)+Integer.parseInt(b);
    }
}
